cg_hermes/deliverables.py

```diff
@@ -76,10 +77,15 @@ def set_pipeline_specific_variables(self):
             self.files = self.get_mutant_files()
             self.configs = Deliverables.build_internal_tag_map(MUTANT_COMMON_TAGS)
         elif self.pipeline == Pipeline.MIP_DNA:
-            LOG.info("Parsing deliverables for mip")
+            LOG.info("Parsing deliverables for mip_dna")
```

Comment:
Could you remove this and add one logging message after the if else:

```python
LOG.info("Parsing deliverables for %s", self.pipeline)
```

Reply:
Done